sources/DropZoneTab.cpp

```diff
@@ -162,19 +166,6 @@ DropZone::MessageReceived(BMessage* msg)
 			be_roster->Launch(kFilerSignature);
 			break;
 		}
-		case B_ABOUT_REQUESTED:
-		{
-			BAboutWindow* about = new BAboutWindow("Filer", kFilerSignature);
-			about->AddDescription(
-				"Filer is an automatic file organizer. It takes the "
-				"files it's opened with or that are dropped on it and moves, "
-				"renames, copies or does all sorts of other things with them "
-				"according to rules created by the user.");
-			about->AddCopyright(2008, "DarkWyrm");
-			about->AddCopyright(2016, "Humdinger, Pete Goodeve");
-			about->AddCopyright(2017, "Owen Pan");
-			about->Show();
-		}
 		default:
```

Comment:
The B_ABOUT_REQUESTED  of the DropTone is still needed. When replicated, the context menu offers a "About Filer" that should open this window. We could try to reuse the the text from the HelpTab to avoid duplicated text/localization.